app/models/docket.rb

```diff
@@ -7,19 +7,24 @@ def docket_type
     fail Caseflow::Error::MustImplementInSubclass
   end
 
-  def appeals(priority: nil, ready: nil)
+  def appeals(priority: nil, ready: nil, genpop: nil, judge: nil)
```

Comment:
Could this be simplified to:
```suggestion
  def appeals(priority: nil, ready: nil, judge: nil)
    genpop = judge.nil?
```
Then line 18 simplifies to:
```ruby
    scope = genpop ? scope.genpop : scope.non_genpop_for_judge(judge)
```

Reply:
It ended up being a lot more painful. :( We needed both.